src/System.Net.Http/tests/FunctionalTests/HttpRequestMessageTest.cs

```diff
@@ -251,6 +251,35 @@ public async Task HttpRequest_BodylessMethod_NoContentLength(string method)
             }
         }
 
+        [Fact]
+        public async Task HttpRequest_BodylessMethod_LargeContentLength()
+        {
+            if (IsWinHttpHandler || IsNetfxHandler || IsUapHandler)
+            {
+                // Some platform handlers differ but we don't take it as failure.
```

Comment:
What do they do in this case?  And I see you've not called out CurlHandler... does that mean it passes your test?

Reply:
Oh, I actually copied the test method from the one above, `HttpRequest_BodylessMethod_NoContentLength()` and wasn't sure about those lines.  Let me just remove them.